### PR TITLE
UI and Layer Updates

### PR DESCRIPTION
**Pull request by sephcoster**
* Implemented new vector base layer for plurality (50 states)
* Updated to show "peers of" text when peers selected
* Added links to peer institution names to pull up same map with their LAR count
* Repaired Print view to print affiliates and peers when options selected.

### CODE REVIEW

File: mapusaurus/mapping/templates/map.html [hunk 1]
```diff
@@ -95,7 +94,7 @@
             PctBlack: L.tileLayer('https://{s}.tiles.mapbox.com/v4/cfpb.FL_TRACT_per-blk_US/{z}/{x}/{y}.png?access_token=' + L.mapbox.accessToken, {errorTileUrl: '/static/basestyle/img/0.png'}),
             PctAsian: L.tileLayer('https://{s}.tiles.mapbox.com/v4/cfpb.FL_TRACT_per-asn_US/{z}/{x}/{y}.png?access_token=' + L.mapbox.accessToken, {errorTileUrl: '/static/basestyle/img/0.png'}),
             PctNonWhite: L.tileLayer('https://{s}.tiles.mapbox.com/v4/cfpb.FL_TRACT_pct_nhwht_US/{z}/{x}/{y}.png?access_token=' + L.mapbox.accessToken, {errorTileUrl: '/static/basestyle/img/0.png'}),
-            Plurality: L.tileLayer('https://{s}.tiles.mapbox.com/v4/cfpb.FL_TRACT_minor_plurality_US/{z}/{x}/{y}.png?access_token=' + L.mapbox.accessToken, {errorTileUrl: '/static/basestyle/img/0.png'}),
+            Plurality: L.tileLayer('https://{s}.tiles.mapbox.com/v4/cfpb.1bd47f29/{z}/{x}/{y}.png?access_token=' + L.mapbox.accessToken, {errorTileUrl: '/static/basestyle/img/0.png'}),
```
Comment (mehtadev17) [inline mapusaurus/mapping/templates/map.html]: What changed here?

**Comment by bsjoberg** [comment]
Damn. That is a way to start the day. Nice work.

File: frontend/src/js/helpers.js [hunk 2]
```diff
@@ -270,6 +273,18 @@
         $('#printLink').attr('href', '/map/print' + window.location.search + window.location.hash );
     }
 
+    function getPeerLinks(){
+        var peerList = $('.peer-link');
+        console.log('peer list: ', peerList);
+        $.each(peerList, function(i, val){
+            console.log("i: ", i);
+            console.log("val: ", val);
```
Comment (mehtadev17) [inline frontend/src/js/helpers.js]: Are these necessary?

**Comment by mehtadev17** [comment]
:+1:

**Pull request merged**